start_kinesis_request.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
What's this file?

Reply:
This is the CLI input used for starting the Kinesis application via the following command:
`aws kinesisanalyticsv2 start-application --cli-input-json file://start_kinesis_request.json`
If we don't want it, I don't mind deleting. Just thought it might be useful to have.

Reply:
Ah, that does seem useful! I think you mentioned that the application doesn't start automatically when the cloudformation stack is deployed?

If you wanted to take it to the next level, you could add a [custom lambda-backed resource](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-cfn-customresource.html)